momepy/functional/_elements.py

```diff
@@ -252,13 +261,22 @@ def _tess(ix, poly, blg, threshold, shrink, segment, enclosure_id):
             return_input=False,
             as_gdf=True,
         )
+        if to_simplify:
+            from shapely import coverage_simplify
+
+            tess.geometry = coverage_simplify(
```

Comment:
```suggestion
            tess.geometry = shapely.coverage_simplify(
```

We do import shapely on top